enverus-ea/parquetjs
# [PRS-25353] - build npm package

Problem
=======
We would like an npm package to be built when changes are merged into main
Solution
========
* Added `azure-pipelines.yml`
* Added some test reporting tools, and `test:ci` and `version` script to help the build

Steps to Verify:
----------------
1. Run the build, and ensure it succeeds
1. Once merged into main, it should push a package to our registry
